readme.md

```diff
@@ -1,5 +1,5 @@
 # git demo
 
-There is tpyo in this file. Help fixing it by forking the repository, modifying the file and sending a pull-request!
+There is typo in this file. Help fixing it by forking the repository, modifying the file and sending a pull-request!
```

Comment:
```suggestion
There is typo in this file. Help fixing it by forking the repository, modifying the file and sending a pull-request! Whatever.
```

I think it make sense to add whatever here. Because its Friday.

Reply:
I doont think so